docs/tutorial/gatsby-image-tutorial/gatsby-image-tutorial.md

```diff
@@ -0,0 +1,361 @@
+
+> _Note: once `gatsby-image` has been installed, it does not need to be included in the `gatsby-config.js` file._
+
+```javascript:title=gatsby-config.js
+plugins: [`gatsby-transformer-sharp`, `gatsby-plugin-sharp`]
+```
+
+## Gatsby-image configuration
+
+Now you're set up to use `gatsby-image`.
+
+### Step 3
+
+Determine where your image files are located. In this example they're in `src/data`.
+
+If you haven't already, make sure that your project is set up to see content inside that directory. That means doing two things:
+
+1.  Install `gatsby-source-filesystem`. Note: If you created your project using `gatsby new <name>`, this first step is already done for you.
+
+```bash
+npm install gatsby-source-filesystem
+```
+
+2. The next step is to make sure your `gatsby-config.js` file specifies the correct folder. In this example it would look like this:
+
+```javascript:title=gatsby-config.js
+plugins: [
+  `gatsby-transformer-sharp`,
+  `gatsby-plugin-sharp`,
+  { resolve: `gatsby-source-filesystem`, options: { path: `./src/data/` },
+]
+```
+
+Now you're ready to start working with `gatsby-image`!
+
+## Step 4
+
+The next step can vary depending on what you're trying to accomplish.
+
+## Querying data for a single image
+
+Use `graphql` to query an image file directly. You can include the relative path to the image file and determine how you want `gatsby-image` to process the file.
+
+```jsx:title=src/pages/index.js
+export const query = graphql`
+  query {
+    file(relativePath: { eq: "headers/headshot.jpg" }) {
+      childImageSharp {
+        fixed(width: 125, height: 125) {
+          ...GatsbyImageSharpFixed
+        }
+      }
+    }
+  }
+`
+```
+
+There are a couple of things to note here.
+
+### Relative image paths and `gatsby-config.js`
+
+You might expect the relative path to be relative to the file the code sits in, in this case that's index.js. However, that doesn't work. The relative path is actually based on the line of code you put in the `gatsby-source-filesystem` config, which points to `src/data`.
+
+### Image fragments
+
+Another thing to note about this query is how it uses the fragment `GatsbyImageSharpFixed` to return a fixed width and height image. You could also use the fragment `GatsbyImageSharpFluid` which produces scalable images that fill their container instead of fitting specific dimensions. In `gatsby-image`, _fluid_ images are meant for images that don’t have a finite size depending on the screen, where as other images are _fixed_.
+
+The query will return a processed image at the highest depth. That is, the deepest part of the return object. The returned result will be automatically passed into the component and attached to the `data` prop. You can then display the image using JSX to automatically output responsive, highly performant HTML.
```

Comment:
```suggestion
The query will return a data object including the processed image in a format usable by the `gatsby-image` component. The returned result will be automatically passed into the component and attached to the `data` prop. You can then display the image using JSX to automatically output responsive, highly performant HTML.
```

I think this part will confuse folks–so I'd recommend rewording it slightly. An alternative suggestion would be to show more of the data object returned so the depth part is less abstract.